eva-accession-pipeline/src/main/resources/monitoring/monitor_duplicate_accessions.py

```diff
@@ -23,7 +23,9 @@
 
 def export_mongo_accessions(mongo_connection_properties, collection_name, export_output_filename):
     export_command = 'mongoexport --host {0} --port {1} --db {2} --username {3} --password {4} ' \
-                     '--authenticationDatabase={5} --collection {6} --type=csv --fields accession -o "{7}" 2>&1' \
+                     '--authenticationDatabase={5} --collection {6} --type=csv --fields accession ' \
+                     "--query  '{{remapped_from: {{$exists: false}}}}' " \
```

Comment:
```suggestion
                     "--query  '{remapped_from: {$exists: false}}' " \
```

Reply:
The brackets have to be doubled to avoid python misinterpreting them as variable placeholders.